pkg/lib/listing.less

```
@@ -180,12 +186,26 @@ tbody.open td.listing-ct-toggle i:before {
 }
 
 /* Listing items have decent padding ... */
-tr.listing-ct-item td,
-tr.listing-ct-item th {
+.listing-ct > thead > tr > th,
+tr.listing-ct-item > th,
+tr.listing-ct-item > td {
     padding: @listing-ct-padding * 1.5;
     vertical-align: text-top; /* baseline doesn't support wrapped lines */
 }
 
+tr.listing-ct-item > td {
+    > .btn {
+        /* .btn has padding of 6px and 1px border */
+        margin-top: -7px;
+        margin-bottom: -7px;
+    }
+    > .btn-control-ct {
+        /* Cockpit's flavor embeggins the button by 8px */
```

What does this code comment mean?
Is _embiggins_ a Quiddish thing? https://wordability.net/tag/embeggin/
Flavor of what?

---

Embiggen is an official word in the Merriam Websters dictionary meaning "to make larger".
https://www.merriam-webster.com/dictionary/embiggen

It's originally from The Simpsons, from a 1996 episode which also coined the word "cromulent" (meaning "perfectly acceptable").

Since then, "A noble spirit embiggens the smallest man" has been added to the intro, at the bottom of the Jebediah Springfield statue. (Probably added when they went to HD? I don't know exactly.)

I accidentally swapped the e and i. Oops.

---

It should be:

```suggestion
        /* Cockpit's flavor embiggens the button by 8px */
```

It's an easter egg in the source, basically. :wink:

---

Changed to:
```css
        /* Adapt for Cockpit's modified button, which is larger by 8px */
```